containers/Connector/Connector.tsx

```diff
@@ -70,12 +72,14 @@ const useConnector = () => {
 	}, [signer, dispatch]);
 
 	useEffect(() => {
-		const rateInterval = setInterval(() => {
-			dispatch(setExchangeRates());
-		}, 15000);
+		const ratesListener = (exchangeRates: Rates) => {
```

Comment:
Still not sure this is the right location for this, seems out of place in the connector. Maybe we can create a new component called AppData or something, as there will be quite a bit of app wide data we need to manage, similar to how we have RefetchProvider now, just doesn't need to be a context but something that lives at the very top level.

Reply:
Created an AppData component, that simply returns its children, but initializes the necessary hooks. It feels like an antipattern though. I hope we can switch to simply running those functions outside components in the future, because putting them inside is just overhead for the component tree.